jwst/associations/lib/dms_base.py

```diff
@@ -358,6 +358,10 @@ def is_item_tso(self, item, other_exp_types=None):
         if item['pntgtype'] != 'science':
             return False
 
+        # Target acquisitions are never TSO
+        if item['exp_type'] in ACQ_EXP_TYPES:
+            return False
+
```

Comment:
Might as well go ahead and add 'NRS_VERIFY' to the list of `ACQ_EXP_TYPES` for JP-1423 and get it over with now.